app/views/external_users/claims/offence_details/_fee_reform_fields.html.haml

```diff
@@ -45,7 +45,7 @@
         ),
         %br
         please visit the
-        %a{ href: "https://www.gov.uk/government/publications/crown-court-fee-guidance", target: '_blank' } Crown Court fee guidance
+        %a{ href: "https://www.gov.uk/government/publications/crown-court-fee-guidance" } Crown Court fee guidance
```

Comment:
Would it not be better to add an accessible warning and open in a new tab/window?

Leaving the current page to read the guidance could lead users to lose data inputted prior to leaving.